admin/client/src/styles/legacy/_style.scss

```diff
@@ -1613,28 +1613,37 @@ form.member-profile-form {
 	}
 }
 
-.cms .cms-panel-padded {
-  &.ReportAdmin {
-    padding: $grid-gutter-width-half;
-    width: 100% !important;
+// Custom styles for ReportAdmin
+.ReportAdmin {
 
-    > .message {
-      margin-bottom: $grid-gutter-width-half;
-    }
+  > fieldset {
+    margin-top: -$grid-gutter-width-half;
 
-    > fieldset {
-      margin-top: -$grid-gutter-width-half;
+    // If form fields are at the top (typically .grid-field) then add additional space
+    > .form-group:first-child {
+      margin-top: $grid-gutter-width-half;
     }
+  }
 
-    .ss-gridfield-buttonrow {
-      margin-left: -20px;
-      margin-right: -20px;
-    }
+  > .message {
+    margin-bottom: $grid-gutter-width-half;
   }
 
-	.cms-content-view {
-		padding: 0 $grid-gutter-width-half $grid-y*1.5;
-	}
+  .toolbar--content {
+    margin-left: -$grid-gutter-width-half;
+    margin-right: -$grid-gutter-width-half;
+  }
+
+  &.cms-panel-padded {
```

Comment:
You could but I think the alternative is worse, if you take a look at the _broken links report_ you will see it has no actions toolbar at the top meaning fields are touching the top. We can't add spacing above the :first-child as it sometimes applies it to hidden messages and fields  and it's hard to tell if they are active or not. 

This probably wouldn't be the way I'd do it if I could adjust all of the HTML but at the moment each section of the ReportAdmin & CMS has completely different markup structure (which we will need to standardise) making it hard to maintain.

The Way I'm approaching this only the first toolbar whether it is part of a gridfield will be negatively spaced, everything else will have appropriate spacing above and to the sides. This means no matter what the panel contains it should look okay.